lib/DDGC/DB/Result/Topic.pm

```diff
@@ -30,7 +30,7 @@ unique_column name => {
         is_nullable => 0,
 };
 
-has_many 'instant_answer_topics', 'DDGC::DB::Result::InstantAnswer::Topics', 'topics_id', {on_delete => 'cascade'};
+has_many 'instant_answer_topics', 'DDGC::DB::Result::InstantAnswer::Topics', 'topics_id';
```

Comment:
@jbarrett why did you have to remove `{on_delete => 'cascade'}`? I mean, I understand you got that error shown, but I don't fully understand why